src/App.js

```diff
@@ -1,12 +1,58 @@
-import React from 'react';
-import './App.css';
-
-function App() {
-  return (
-    <div className="App">
-      <h1>Fast Fingers </h1>
-    </div>
-  );
+import React, { Component } from "react";
+import Landing from "./pages/Landing";
+import Game from "./pages/Game";
+
+const difficulties = {
+  easy: {
+    key: "easy",
+    label: "Easy",
+    factor: 1,
+  },
+  medium: {
+    key: "medium",
+    label: "Medium",
+    factor: 1.5,
+  },
+  hard: {
+    key: "hard",
+    label: "Hard",
+    factor: 2,
+  },
+};
```

Comment:
I think this can be moved into some constants file, so that we can configure it later. easily

Reply:
I mean outside main code.

Reply:
Yes, I will move it to a separate constants.js file